zanata-war/src/main/java/org/zanata/action/ProjectSlug.java

```diff
@@ -22,16 +22,18 @@
 
 import java.io.Serializable;
 
-import org.apache.deltaspike.core.api.scope.ViewAccessScoped;
+import org.apache.deltaspike.core.api.scope.ConversationGroup;
+import org.apache.deltaspike.core.api.scope.GroupedConversationScoped;
 
 /**
- * We set this to ViewAccessScoped so that it can retain the value accross
- * requests (original request, subsequent ajax requests)
+ * We set this to GroupedConversationScoped so that it can retain the value
+ * across requests (original request, subsequent ajax requests)
```

Comment:
I think I found ViewScoped to be a little fragile on some of these pages,
possibly because of the autocomplete callbacks.

Are you suggesting a different comment?

On 1 March 2016 at 09:52, Carlos Munoz <notifications@github.com> wrote:

> In zanata-war/src/main/java/org/zanata/action/ProjectSlug.java
> <https://github.com/zanata/zanata-server/pull/1072#discussion_r54500884>:
>
> >
> >  /**
> > - * We set this to ViewAccessScoped so that it can retain the value accross
> > - * requests (original request, subsequent ajax requests)
> > + * We set this to GroupedConversationScoped so that it can retain the value
> > + * across requests (original request, subsequent ajax requests)
>
> wouldn't @ViewScope solve this problem? @GroupedConversationScoped should
> solve this problem across multiple requests for different pages.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zanata/zanata-server/pull/1072/files#r54500884>.
>



-- 
Sean Flanigan

Principal Software Engineer
Globalisation Tools Engineering
Red Hat